polyfills/Event/config.json

```diff
@@ -9,7 +9,7 @@
 	"browsers": {
 		"firefox": "6 - 10",
 		"ie": "6 - 11",
-		"ie_mob": "10",
+		"ie_mob": "11",
```

Comment:
If anything, I'd say `<12` because we prefer `<` syntax to `<=`.  I don't see a record of `Event` in `ie_mob` in the compat data, maybe it's fallen out of the 30-day window.